efi-log.ks.in

```diff
@@ -0,0 +1,36 @@
+#version=DEVEL
+#test name: efi-log
+%ksappend repos/default.ks
+network --bootproto=dhcp
+
+bootloader --timeout=1
```

Comment:
I think other than the %post script this kickstart tests does not really need anything special, right ? In that case I would suggest using of the "common boilerplate" kickstart fragments instead to inject the common things. For example this one:

https://github.com/rhinstaller/kickstart-tests/blob/master/fragments/shared/common/common_no_payload.ks

A similar (mainly needs to do stuff in %post) kickstart tests that uses this fragment:

https://github.com/rhinstaller/kickstart-tests/blob/2ca0796cd976eb7561fef361f0830005b2196398/ui_cmdline.ks.in#L6

Reply:
Updated according to suggestion.